test/src/main/java/org/jvnet/hudson/test/HudsonTestCase.java

```diff
@@ -276,6 +277,8 @@
      */
     protected File explodedWarDir;
 
+    private boolean origDefaultUseCache = true;
```

Comment:
I generally do not bother patching `HudsonTestCase` any more, especially after most of its usages have already been fixed in core to be `JenkinsRule`, but since you have already done the work…